app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -61,9 +61,34 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
   const resultsContextValue: ResultsContextType = {
     analysis,
     sortedBranchNames,
-    controlBranchName: sortedBranchNames[0],
+    controlBranchName:
+      sortedBranchNames.length > 0 ? sortedBranchNames[0] : undefined,
   };
 
+  const controlBranchError =
+    sortedBranchNames.length === 0 ? (
+      <AnalysisErrorAlert
+        errors={[
+          {
+            metric: null,
+            message: "Analysis could not be performed",
```

Comment:
Maybe be a little more specific here about the problem being that no control branch was found?

Reply:
Oh yea, I meant to update that, thanks for the reminder! Updated in new commit.